continuous_integration/install.sh

```diff
@@ -9,6 +9,7 @@ if [[ "$CC_OUTER_LOOP" == "clang-10" || "$CC_INNER_LOOP" == "clang-10" ]]; then
     wget https://apt.llvm.org/llvm.sh
     chmod +x llvm.sh
     sudo ./llvm.sh 10
+    sudo apt-get install libomp-dev
```

Comment:
I don't really understand this addition. Clang doesn't come with the LLVM OpenMP runtime ?

Reply:
Not anymore apparently.

Reply:
Or at least `llvm.sh` does not install this package by default anymore.